src/dragonfs.c

```diff
@@ -684,36 +703,93 @@ int dfs_dir_findnext(char *buf)
     return __dfs_findnext(buf, &next_entry);
 }
 
+static uint32_t prime_hash(const char *str, uint32_t prime)
+{
+    uint32_t hash = 0;
+    while(*str) {
+        char c = *str++;
+        hash = (hash * prime) + c;
+    }
+    return hash;
+}
 
-int dfs_open(const char * const path)
+static int32_t lookup_file_hash_entry(uint32_t hash)
```

Comment:
Add a comment to this function to explain what it does, specifically the fact that in case of conflicts it will return the first index

Reply:
Function no longer exists.